charts/apisix/templates/pdb.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if (and .Values.apisix.enabled .Values.apisix.podDisruptionBudget.enabled) }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "apisix.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "apisix.labels" . | nindent 4 }}
+spec:
+{{- if .Values.apisix.podDisruptionBudget.minAvailable }}
+  minAvailable: {{ .Values.apisix.podDisruptionBudget.minAvailable }}
+{{- end }}
+{{- if .Values.apisix.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ .Values.apisix.podDisruptionBudget.maxUnavailable }}
+{{- end }}
```

Comment:
`minAvailable` and `maxUnavailable` should be set exclusively, so here use a `if-else` block will be better.

Reply:
good idea.